docs/COMPILATION.MD

```diff
@@ -189,7 +189,34 @@ Once set up you can run the GUI interface from the terminal `./ccextractorGUI`
 
 ## Windows
 
-Open the windows/ccextractor.sln file with Visual Studio (2015 at least), and build it.
+**Using Visual Studio**
+
+#### Here you can see two files .vcxproj(marked with blue arrows) and one .sln project(marked with a red arrow)
+
+![alt text](img/projectFiles.png)
+
+You have to open .sln if you want to build both non GUI and GUI ccextractor or you can build them seperately opening .vcxproj files one by one.
```

Comment:
```suggestion
Using Visual Studio (2015 or above), open `ccextractor.sln`. This will build both CCExtractor and its GUI. To build them separately, open the respective `.vcxproj` file.
```